website/src/pages/label/label_prompter_reply.tsx

```diff
@@ -1,32 +1,9 @@
-import Head from "next/head";
-import { TaskEmptyState } from "src/components/EmptyState";
 import { getDashboardLayout } from "src/components/Layout";
-import { LoadingScreen } from "src/components/Loading/LoadingScreen";
-import { Task } from "src/components/Tasks/Task";
-import { useLabelPrompterReplyTask } from "src/hooks/tasks/useLabelingTask";
+import { TaskPage } from "src/components/TaskPage/TaskPage";
+import { TaskType } from "src/types/Task";
 export { getDefaultStaticProps as getStaticProps } from "src/lib/default_static_props";
 
-const LabelPrompterReply = () => {
-  const { tasks, isLoading, trigger, reset } = useLabelPrompterReplyTask();
-
-  if (isLoading) {
-    return <LoadingScreen />;
-  }
-
-  if (tasks.length === 0) {
-    return <TaskEmptyState />;
-  }
-
-  return (
-    <>
-      <Head>
-        <title>Label Prompter Reply</title>
```

Comment:
Where did these title and description texts go? I don't see locale files with these strings.

Reply:
Thank you @fozziethebeat, I understand and appreciate where you're both coming from.  The reason this change also includes a refactor was to prevent from adding error handing code for no available tasks in ten different files, now we have it in just one.  In terms of the title and description we are able to leverage the already existing `label` and `desc` properties from the locale files to apply to the `meta` tags in the header.  I hope that helps you understand the change more clearly?  Any problems please let me know.

https://github.com/LAION-AI/Open-Assistant/blob/main/website/public/locales/en/tasks.json